spinta/utils/passwords.py

```diff
@@ -0,0 +1,52 @@
+import base64
+import hashlib
+import hmac
+import math
+import os
+import random
+
+
+def _bencode(value: bytes) -> str:
+    return base64.urlsafe_b64encode(value).decode().rstrip('=')
```

Comment:
Why are we removing '=' characters at the end of the string?

That will explain re-adding them in line 14

Reply:
Mainly because of esthetic reasons and because python does not implement full base64 standard. Padding in base64 URL safe encoding is not required if there is no need to concatenate multiple base64 encoded strings. So in this case password hashes will never be concatenated.

https://tools.ietf.org/html/rfc4648#section-5